RobotConstantsBase.java

```diff
@@ -0,0 +1,167 @@
+package frc.team670.mustanglib;
+
+import static java.util.Map.entry;
+
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+import java.util.Map;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.auto.PIDConstants;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.math.trajectory.constraint.DifferentialDriveKinematicsConstraint;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.SerialPort;
+import frc.team670.mustanglib.subsystems.drivebase.SwerveDrive;
+import frc.team670.mustanglib.swervelib.Mk4iSwerveModuleHelper.GearRatio;
+import frc.team670.mustanglib.swervelib.ModuleConfiguration;
+import frc.team670.mustanglib.swervelib.SdsModuleConfigurations;
+/**
+ * A place for storing constants required in mustanglib. 
+ * This includes most drivebase constants. For every robot, add mac address like below as well as relevant constants
```

Comment:
Could we please talk before you make more changes? Maybe I'm not understanding what's happening but if it's what it looks like, I don't think we're going in the right direction.

I also don't understand why we're changing all the naming to hungarian notation. It's a bunch of changes for no appreciable benefit and it breaks common java style, certainly for constants.

Reply:
Read some more and maybe it's just in a partial state. Still, I'd like to talk to understand where you're trying to go. And the comment about all the renaming stands.